apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Generated by `createservice property.RegisterResidentService --type mutations`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('../access/RegisterResidentService')
+const { Resident } = require('../utils/serverSchema/index')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+
+const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
```

Comment:
Wow, we have resident as user type and resident as Entity. It's confusing

Reply:
@dkoviazin added `User.type` field with value `resident`, which will distinguish mobile clients from "resident" app.
Yeah, it's confusing ;)
I will explain it in more detail in SBERDOMA-878